octobot_evaluators/channels/evaluators.py

```diff
@@ -96,6 +97,7 @@ async def new_consumer(self,
 
     def get_filtered_consumers(self,
```

Comment:
Why filter by data ? To me it doesn't make any sense

Reply:
I added it to be able to do it later without thinking more about it, if it has an impact on performances, we should remove it since it's not used for now

Reply:
I really don't understand when we'll want to filter by data ?

Reply:
when resetting TA, not to up anything in the queue of other evaluators for example

Reply:
I don't think filtering on a dict for a specific key is a good practice. If you think you need something like an "order_name" to filter on I think it's better to add a new string param than a dict. It would be something like the `state` for Mode channel

Reply:
Since its not required for now let's just remove this filter and we will add arguments later if we need to, ok for you ?

Reply:
ok :+1: